### DISCUSSION TITLE
Comment appears empty after editing

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#84948] [Mobile] [answered]
1. Open GitHub Mobile beta for Android
2. Visit https://github.com/github/rest-api-description/issues/3259 via a notification
3. Scroll to https://github.com/github/rest-api-description/issues/3259#issuecomment-1865096737
4. Trigger the editor
5. Replace the text with a single line
6. Apply
7. Scroll to see the comment

### Actual Results
![Screenshot_20240108-075515.png](https://github.com/community/community/assets/2119212/19d87e39-98dd-4397-b6d4-4a0e4c114787)

Note that it eventually healed, but it should never have appeared like this.

**Comment by elisealix22** [comment]
Thanks for reporting this @jsoref!  I'm having a bit of trouble reproducing and would love to learn more. It seems like in your report, the comment was offscreen before scrolling back to it and it appearing empty. Was it offscreen when you edited it? Or was it always onscreen and when you returned from the editor, it appeared blank?

**Reply by jsoref** [reply]
I think it was on screen in both cases. I definitely scrolled up/down to confirm it was missing. I tried editing without saving to confirm the content wasn't truly gone. It only came back when I revisited the discussion.

**Reply by elisealix22** [reply, marked as answer]
Wow okay, thank you for the clarification! I'll open up an issue and try with the team to reproduce and resolve. Sorry you're running into this.
